doc/usage.md

```diff
@@ -0,0 +1,54 @@
+# Quick start
```

Comment:
# Quick start

## Configure
Example where the priviliges are defined in the main.js using an object
literal.

```js
  .plugin('aurelia-acl', {
    chat      : ['read', 'post'],
    product   : ['read'],
    user      : ['read'],
    statistics: ['*'] // has acces to all statistics
  });
```

## Attribute

```html
<messages allowed="{chat: ['read', 'post']"></messages>>
```
## View model

Implement acl in your view model.

```js
import {Acl} from 'aurelia-acl';

@inject(Acl)
export class Page {
  constructor(acl) {
    this.isAllowed = acl.isAllowed;
  }
}
```

## View
Now tell the view to only render the element if has the `read` and `post`.

```html
  <messages if.bind="isAllowed({chat: ['read', 'post'])"></messages>
```